src/serializers/encoder/v11/encode_schema.c

```diff
@@ -49,7 +49,18 @@ static inline void _RdbSaveIndexData(RedisModuleIO *rdb, Index *idx) {
 	RedisModule_SaveUnsigned(rdb, fields_count);
 	for(uint i = 0; i < fields_count; i++) {
 		// Indexed property
-		RedisModule_SaveStringBuffer(rdb, idx->fields[i], strlen(idx->fields[i]) + 1);
+		RedisModule_SaveStringBuffer(rdb, idx->fields[i].name, strlen(idx->fields[i].name) + 1);
+		if(idx->type == IDX_FULLTEXT) {
+			RedisModule_SaveDouble(rdb, idx->fields[i].weight);
+			RedisModule_SaveUnsigned(rdb, idx->fields[i].nostem);
+			if(idx->fields->phonetic) {
+				RedisModule_SaveStringBuffer(rdb, idx->fields[i].phonetic, 
+					strlen(idx->fields[i].phonetic) + 1);
+			} else {
+				RedisModule_SaveStringBuffer(rdb, "no", 
```

Comment:
Can `no` be the default value for `phonetic` ? in which case we can simply change this to:
`RedisModule_SaveStringBuffer(rdb, idx->fields[i].phonetic, strlen(idx->fields[i].phonetic) + 1);`